package.json

```diff
@@ -1,11 +1,18 @@
 {
   "scripts": {
     "test": "xo && ava",
+    "build": "cd src && each '{*.js,!(libs)/**/*.js}' --parallel 10 -- rollup --config node:webext $PWD/%p --output $PWD/../extension/%p",
```

Comment:
`--parallel 10` => `--parallel=4` I'm not seeing any better perf after 4. Actually slightly less.

`!(libs)/**/*.js` => `!libs/vendor/*.js` to be explicit